### PR TITLE
fix: deleting "index.html" in some links

### PR DESCRIPTION
**Pull request by Malaaaa**

### CODE REVIEW

**Review by Malaaaa — COMMENTED** [review]
no confict

File: files/zh-cn/learn/getting_started_with_the_web/how_the_web_works/index.html [hunk 1]
```diff
@@ -89,12 +89,12 @@ <h2 id="到底发生了什么？">到底发生了什么？</h2>
...
+<p>当浏览器向服务器发送请求获取 HTML 文件时，HTML 文件通常包含 "link" 和 "script" 元素，这些元素分别指向了外部的 <a href="../../css">CSS</a> 样式表文件和 <a href="../../javascript">JavaScript</a> 脚本文件。了解这些文件被<a href="../../../web/performance/how_browsers_work">浏览器解析</a>的顺序是很重要的：</p>
 
 <ul>
     <li>浏览器首先解析 HTML 文件，并从中识别出所有的 "link" 和 "script" 元素，获取它们指向的外部文件的链接。</li>
     <li>继续解析 HTML 文件的同时，浏览器根据外部文件的链接向服务器发送请求，获取并解析 CSS 文件和 JavaScript 脚本文件。</li>
-    <li>接着浏览器会给解析后的 HTML 文件生成一个 <a href="../../../web/api/document_object_model/index.html">DOM</a> 树（在内存中），会给解析后的 CSS 文件生成一个 <a href="../../../glossary/cssom/index.html">CSSOM</a> 树（在内存中），并且会<a href="../../../web/performance/how_browsers_work/index.html">编译和执行</a>解析后的 JavaScript 脚本文件。</li>
+    <li>接着浏览器会给解析后的 HTML 文件生成一个 <a href="../../../web/api/document_object_model">DOM</a> 树（在内存中），会给解析后的 CSS 文件生成一个 <a href="../../../glossary/cssom">CSSOM</a> 树（在内存中），并且会<a href="../../../web/performance/how_browsers_work">编译和执行</a>解析后的 JavaScript 脚本文件。</li>
```
Comment (Malaaaa) [inline files/zh-cn/learn/getting_started_with_the_web/how_the_web_works/index.html:97]: delete 'index.html' to make url works

File: files/zh-cn/learn/getting_started_with_the_web/how_the_web_works/index.html [hunk 2]
```diff
@@ -89,12 +89,12 @@ <h2 id="到底发生了什么？">到底发生了什么？</h2>
 
 <h2 id="解析组成文件的顺序">解析组成文件的顺序</h2>
 
-<p>当浏览器向服务器发送请求获取 HTML 文件时，HTML 文件通常包含 "link" 和 "script" 元素，这些元素分别指向了外部的 <a href="../../css/index.html">CSS</a> 样式表文件和 <a href="../../javascript/index.html">JavaScript</a> 脚本文件。了解这些文件被<a href="../../../web/performance/how_browsers_work/index.html">浏览器解析</a>的顺序是很重要的：</p>
+<p>当浏览器向服务器发送请求获取 HTML 文件时，HTML 文件通常包含 "link" 和 "script" 元素，这些元素分别指向了外部的 <a href="../../css">CSS</a> 样式表文件和 <a href="../../javascript">JavaScript</a> 脚本文件。了解这些文件被<a href="../../../web/performance/how_browsers_work">浏览器解析</a>的顺序是很重要的：</p>
```
Comment (Malaaaa) [inline files/zh-cn/learn/getting_started_with_the_web/how_the_web_works/index.html:92]: delete 'index.html' to make url works

File: files/zh-cn/learn/html/introduction_to_html/the_head_metadata_in_html/index.html [hunk 3]
```diff
@@ -135,10 +135,11 @@ <h3 id="实践操作_在搜索引擎中description的使用">实践操作: 在
 <ol>
  <li>访问 <a href="https://developer.mozilla.org/zh-CN/">MDN Web Docs</a>。</li>
  <li>查看网页源代码（<em>通过鼠标右键点击网页在弹出的菜单中选择[查看网页源代码]</em>）</li>
- <li>找到description的meta标签。就和如下展示的这样：
-  <pre class="brush: html notranslate">&lt;meta name="description" content="The Mozilla Developer Network (MDN) provides
-information about Open Web technologies including HTML, CSS, and APIs for both
-Web sites and HTML5 Apps. It also documents Mozilla products, like Firefox OS."&gt;</pre>
+ <li>找到description的meta标签。就和如下展示的这样：+
+  <pre class="brush: html notranslate">&lt;meta name="description" content="The MDN Web Docs site
```
Comment (Malaaaa) [inline files/zh-cn/learn/html/introduction_to_html/the_head_metadata_in_html/index.html:139]: the following picture is about MDN Web Docs

**Comment by Malaaaa** [comment]
Test

**Comment by irvin** [comment]
please don't manually add review request, the bot will call the relative locale team volunteers. Manually add won't make it faster.

**Pull request merged**